src/Tizen.NUI/src/internal/Transition/TransitionGroupItem.cs

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// TransitionGroupItem is an object to set Fade transition of a View that will appear or disappear.
+    /// TransitionGroupItem object is required to be added to the TransitionSet to play.
+    /// </summary>
+    internal class TransitionGroupItem : TransitionItemBase
+    {
+        private List<TransitionItemBase> transitionItemList = null;
+
+        private bool useGroupTimePeriod = false;
+        private bool stepTransition = false;
+        private bool useGroupAlphaFunction = false;
+
+        private View transitionView = null;
+
+        /// <summary>
+        /// Creates an initialized fade.<br />
+        /// </summary>
+        /// <remarks>DurationmSeconds must be greater than zero.</remarks>
+        public TransitionGroupItem(View view, List<TransitionBase> transitionList, bool useGroupTimePeriod, bool stepTransition, bool useGroupAlphaFunction, bool appearingTransition, TimePeriod timePeriod, AlphaFunction alphaFunction)
+        : base(view, appearingTransition, timePeriod, alphaFunction)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            this.useGroupTimePeriod = useGroupTimePeriod;
+            this.stepTransition = stepTransition;
+            this.useGroupAlphaFunction = useGroupAlphaFunction;
+            transitionView = view;
+
+            transitionItemList = new List<TransitionItemBase>();
+            for (int index = 0; index < transitionList.Count; ++index)
+            {
+                TimePeriod localTimePeriod = new TimePeriod(transitionList[index].GetTimePeriod().DelayMilliseconds, transitionList[index].GetTimePeriod().DurationMilliseconds);
```

Comment:
could you check the warning below?
maybe, this warning can be resolved by adding `localTimePeriod.Dispose();` after completely used.

Reply:
Thank you I fixed my code to do not need to call Dispose().